app/src/main/java/com/ataulm/whatsnext/FilmSummaryViewHolder.kt

```diff
@@ -4,14 +4,27 @@ import androidx.recyclerview.widget.RecyclerView
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.annotation.Px
+import coil.api.load
 import kotlinx.android.synthetic.main.view_film_summary.view.*
 
 class FilmSummaryViewHolder internal constructor(itemView: View) : RecyclerView.ViewHolder(itemView) {
 
+    init {
+        itemView.imageView.clipToOutline = true
+        itemView.backgroundView.clipToOutline = true
```

Comment:
clipToOutline doesn't work from XML - could be a good bug to fix in AOSP. 

also could be a good one to backport to a layout inflater (read this value and then set it programmatically).